sound/soc/sof/ipc4-topology.h

```diff
@@ -173,4 +173,23 @@ struct sof_ipc4_gain {
 	struct sof_ipc4_msg msg;
 };
 
+enum sof_ipc4_mixer_type {
+	sof_ipc4_mix_in,
+	sof_ipc4_mix_out
```

Comment:
do we really need these? if we ever need to, we can always distinguish the 2 with their UUID's right?

Reply:
> do we really need these? if we ever need to, we can always distinguish the 2 with their UUID's right?

Right, lets remove it.